### PR TITLE
Release 4.4.1

### PR DESCRIPTION
**Pull request by SylvainCorlay**
I added myself as a recipe maintainer.

### CODE REVIEW

**Comment by jasongrout** [comment]
Circle-ci says there's a sha256 mismatch:

```
Source cache directory is: /feedstock_root/build_artefacts/src_cache
Downloading source to cache: notebook-4.4.0.tar.gz
Downloading https://files.pythonhosted.org/packages/0f/52/5564e26247b8c17307aabe410bd323cbceb5a2f341ca19ac9ba10d9f107a/notebook-4.4.0.tar.gz
Success
SHA256 mismatch: 'ea889941254bf681419da686ffd2b3226505342bac4a4f4995c6c226acb41883' != '807b83893639895f506a21c44b8b2252c20472500a8b896979e052987c7a01cc'
```

**Comment by SylvainCorlay** [comment]
Hum it worked locally. Checking.

**Comment by SylvainCorlay** [comment]
OK, the culprit is my conda cache.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+{% set version = "4.4.0" %}
+{% set sha256 = "ea889941254bf681419da686ffd2b3226505342bac4a4f4995c6c226acb41883" %}
 
 package:
   name: notebook
   version: {{ version }}
 
 source:
   fn: notebook-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/n/notebook/notebook-{{ version }}.tar.gz
```
Comment (minrk) [inline recipe/meta.yaml]: The pypi.io URL should be working now.

Comment (SylvainCorlay) [inline recipe/meta.yaml]: nope

**Comment by minrk** [comment]
4.4.1 is up now with checksum: dfadef2babd7c04c6c257df7d07d7ba587e503dbb4e4c95305f9a95b8d3a9765

**Comment by minrk** [comment]
Circle passed, and I don't imagine the others will show anything new for such a tiny release, so I'll go ahead and merge.

**Pull request merged**